projects/igniteui-angular/src/lib/slider/slider.component.spec.ts

```diff
@@ -385,39 +396,74 @@ describe('IgxSlider', () => {
+            pending('Investigate deeper why sliding is not performed');
+            const FromTumb = fixture.debugElement.query(By.css(TUBM_FROM_CLASS)).nativeElement;
+            slider.continuous = true;
+            fixture.detectChanges();
+
+            expect(slider.continuous).toBe(true);
+
+            const sliderEl = fixture.debugElement.query(By.css(SLIDER_CLASS)).nativeElement;
+            sliderEl.dispatchEvent( new Event('pointerdown'));
+            fixture.detectChanges();
+            FromTumb.dispatchEvent(new Event('focus'));
+            fixture.detectChanges();
+
+            expect(sliderEl).toBeDefined();
+            return panRight(sliderEl, sliderEl.offsetHeight, sliderEl.offsetWidth, 200)
+            .then(() => {
+                fixture.detectChanges();
+                const activeToTumb = fixture.debugElement.query(By.css('.igx-slider__thumb-to--active'));
+                const activeFromTumb = fixture.debugElement.query(By.css('.igx-slider__thumb-from--active'));
+                expect(slider.value).toEqual({ lower: 60, upper: 100 });
+                expect(activeToTumb).toBeNull();
+                expect(activeFromTumb).toBeNull();
+                done();
+            });
+        });
+
+        it('should switch from left thumb to be focused upper when lower value is equal to upper', async() => {
             slider.value = {
                 lower: 60,
                 upper: 60
             };
 
             fixture.detectChanges();
 
-            const fromThumb = fixture.nativeElement.querySelector('.igx-slider__thumb-from');
+            const fromThumb = fixture.nativeElement.querySelector(TUBM_FROM_CLASS);
             fromThumb.dispatchEvent(new Event('focus'));
+            await wait();
+            fixture.detectChanges();
+
             UIInteractions.triggerKeyDownEvtUponElem('ArrowRight', fromThumb, true);
+            await wait(50);
             fixture.detectChanges();
 
             expect((slider.value as IRangeSliderValue).lower).toBe(60);
             expect((slider.value as IRangeSliderValue).upper).toBe(60);
-            expect(document.activeElement).toBe(fixture.nativeElement.querySelector('.igx-slider__thumb-to'));
+            expect(document.activeElement).toBe(fixture.nativeElement.querySelector(TUBM_TO_CLASS));
         });
 
-        it('should switch from right thumb to be focused lower when upper value is equal to lower', () => {
+        it('should switch from right thumb to be focused lower when upper value is equal to lower', async() => {
             slider.value = {
                 lower: 60,
                 upper: 60
             };
             fixture.detectChanges();
 
-            const toThumb = fixture.nativeElement.querySelector('.igx-slider__thumb-to');
+            const toThumb = fixture.nativeElement.querySelector(TUBM_TO_CLASS);
             toThumb.dispatchEvent(new Event('focus'));
+            await wait();
```

Comment:
Let make all tests, that perform an operation different from tracking the labels state synchronous. Because we do not perform any async operation under the hood.